src/main/java/com/github/paweladamski/httpclientmock/HttpClientVerifyBuilder.java

```diff
@@ -84,6 +86,33 @@ public HttpClientVerifyBuilder withParameter(String name, Matcher<String> matche
     ruleBuilder.addParameterCondition(name, matcher);
     return this;
   }
+  
+  /**
+   * Request body must contain the given URL-encoded form parameter (typically
+   * found in POST requests). Alternatively, parameters may be specified all at
+   * once using {@link #withBody(Map)}.
```

Comment:
You forgot to change withBody int withFormParameters.

Reply:
Fixed in cb31d06f00a64792e996a5080e783986c22bbb16.